src/interfaces/IPBHKingOfTheHill.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+interface IPBHKotH {
+    function ctf() external;
+    function gameEnd() external view returns (uint128);
```

Comment:
Can we also add `gameStart()`? This will allow players to run their bots ahead of time.